__tests__/app.tests.js

```diff
@@ -36,3 +36,15 @@ describe("/api/topics", () => {
       });
   });
 });
+describe("GET api ", () => {
+  test("GET - API should return a json with all the available end points ", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((res) => {
+        console.log(res.body);
+        expect(typeof res.body).toEqual("object");
+        expect(res.get("content-type")).toMatch(/application\/json/);
+      });
```

Comment:
I think here you've just expected to get Json back, you could make a more specific assertion by requiring in the JSON file in your test file, and expecting the response to be identical to said file.